Create sampleresources.md

garora opened the pull request:
Adding helpful sample.resources. So, users can get all links at one place.

dnfclas commented:
Hi __@garora__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by .NET Foundation and real humans are currently evaluating your PR.</p>

TTYL, DNFBOT;

Eilon commented on sampleresources.md:
```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
```
Typo: `resorces` -> `resources`

garora replied:
Corrected!

Eilon commented on sampleresources.md:
```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+Following(s) are helpful resources from our contributors: 
```
Can remove this whole line.

garora replied:
removed in latest PR

Eilon commented on sampleresources.md:
```diff
@@ -0,0 +1,15 @@
+Following(s) are helpful resources from our contributors: 
+
+##Samples
+- [GitHub](https://github.com/aspnet/WebHooks/tree/master/samples/#)
```
Both of these sample URLs are 404's.

garora replied:
These URLs will be automatically fixed, once current dev branch merged down to master.

Eilon replied:
Ah, I see. What is the `GitHub` link then? Maybe just call it "All samples"? (I interpreted it to mean "A sample for using GitHub web hooks".)

Eilon commented:
I also wonder if this is probably best to put in the main `Readme.md` - that already has a blog post link and it's probably the first place for people to look anyway. Having a separate file I think would reduce the chances of someone easily finding it.

garora commented:
@Eilon - Initially, I was thinking the same. Then I thought to push this new file for review. I am updating these changes in ```Readme.md``` and afterward will delete ```sampleresources.md``` file. I appreciate for support & your time.

garora commented:
@Eilon - Updated ```readme.md``` and removed ```sampleresources.md```.
All links are working and it looks good to me, request you to please review and do merge it down to your dev/master branch, if you find it fit.

Eilon commented:
Was there a bad merge here? The PR looks messed up now, with a bunch of unrelated commits on it...

garora commented:
Not sure, I just updated readme.md and deleted samplresources.md file.
On 27/09/2015 11:18 PM, "Eilon Lipton" <notifications@github.com> wrote:

> Was there a bad merge here? The PR looks messed up now, with a bunch of
> unrelated commits on it...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/WebHooks/pull/6#issuecomment-143580626>.
>

garora commented:
@Eilon- on the otherhand, if you suggest I will make new PR for Updates of. '''Readme.md'''.

Eilon commented:
@garora that might be a good idea. Not sure what happened in this PR.

garora commented:
Sure - I will do it. Thanks for your support and time

On Mon, Sep 28, 2015 at 12:10 PM, Eilon Lipton <notifications@github.com>
wrote:

> @garora <https://github.com/garora> that might be a good idea. Not sure
> what happened in this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/WebHooks/pull/6#issuecomment-143653089>.
>



-- 
Thanks & Regards,
Gaurav Arora

garora commented:
Closed PR. Created new [PR](https://github.com/aspnet/WebHooks/pull/7) .
